app/views/idv/how_to_verify/show.html.erb

```diff
@@ -11,57 +11,69 @@
     ) do |f|
 %>
   <div role="group" aria-labelledby="how-to-verify-info">
-    <%= f.radio_button(
+    <%= f.hidden_field(
           :selection,
-          Idv::HowToVerifyForm::REMOTE,
-          class: 'usa-radio__input usa-radio__input--tile',
+          value: Idv::HowToVerifyForm::REMOTE,
         ) %>
     <%= f.label(
           :selection_remote,
-          class: 'usa-radio__label usa-radio__label--illustrated usa-radio__label-illustrated-large',
         ) do %>
-          <%= image_tag asset_url('idv/remote.svg'),
-                        width: 88,
-                        height: 92,
-                        class: 'usa-radio__image',
-                        alt: t('image_description.laptop_and_phone')
-          %>
-          <div class="usa-radio__label--text">
-            <span class="usa-tag usa-tag--informative">
-              <%= t('doc_auth.tips.most_common') %>
-            </span>
-            <h2 class="h3"><%= t('doc_auth.headings.verify_online') %></h2>
-            <span class="usa-radio__label-description">
-              <p><%= t('doc_auth.info.verify_online_instruction') %></p>
-              <p><%= t('doc_auth.info.verify_online_description') %></p>
-            </span>
+          <div class="grid-row border-bottom border-primary-light">
+            <div classe="tablet:grid-col">
+              <%= image_tag asset_url('idv/remote.svg'),
+                            width: 88,
+                            height: 88,
+                            class: 'margin-right-1',
+                            alt: t('image_description.laptop_and_phone')
+              %>
+            </div>
+            <div class="tablet:grid-col">
+              <h2 class="h3 margin-top-0"><%= t('doc_auth.headings.verify_online') %></h2>
+              <div>
+                <p><%= t('doc_auth.info.verify_online_instruction') %></p>
+                <p><%= t('doc_auth.info.verify_online_description') %></p>
+              </div>
+              <%= f.submit t('forms.buttons.continue_remote'), class: 'display-block margin-y-5' %>
+            </div>
           </div>
         <% end %>
-    <%= f.radio_button(
+  </div>
+<% end %>
+<%= simple_form_for(
+      @idv_how_to_verify_form,
```

Comment:
Is there a way to get these two forms to render with separate id's? The rendered HTML has them with the same id: "new_idv_how_to_verify_form"

Reply:
This sounds like a basic HTML validation thing we'd ideally be checking in some baseline checks like our `expect_page_to_have_no_accessibility_violations` helper. Is this page currently validated with that? Or is that helper not catching duplicate IDs?

Reply:
Seems like we're running that exact helper on line 31 [here](https://github.com/18F/identity-idp/blob/b0c9361ed4b7253123dc5b6ea875d260d8522352/spec/features/accessibility/idv_pages_spec.rb#L20).

I guess that helper isn't catching duplicate IDs?

Reply:
Hm, yeah. Well, doesn't need to block here. We can update the IDs and follow-up separately to see about incorporating an HTML validation check.

Reply:
should we add a follow up ticket?

Reply:
Follow-up to enhance accessibility tests: #10362